src/upload/result.rs

```diff
@@ -36,9 +36,9 @@ pub struct Response {
     pub version: usize,
     pub version_id: String,
     pub signature: String,
-    pub width: usize,
-    pub height: usize,
-    pub format: String,
+    pub width: Option<usize>,
+    pub height: Option<usize>,
+    pub format: Option<String>,
```

Comment:
Do you know if it is documented somewhere?

Reply:
Yeah files like audio files, text files etc don't have width and height. There's a an example [here](https://cloudinary.com/documentation/upload_images#uploading_non_media_files_as_raw_files)

Reply:
Instead of optional width params, I would like a more strict result type. If we upload an image, we know the result should have width, height, and format. And if we upload audio, it does not.